webview/src/plots/components/App.test.tsx

```diff
@@ -856,13 +856,10 @@ describe('App', () => {
 
     dragEnter(plots[1], plots[0], DragEnterDirection.LEFT)
 
-    const plotsWithDropTarget = screen.getAllByTestId(/^plot_/)
-    expect(plotsWithDropTarget.map(plot => plot.id)).toStrictEqual([
-      'plot-drop-target',
-      plots[0].id,
-      plots[1].id,
-      plots[2].id
-    ])
+    const target = screen.getByTestId('drop-target')
+
+    // eslint-disable-next-line testing-library/no-node-access
+    expect(target.nextElementSibling?.id).toStrictEqual(plots[0].id)
```

Comment:
Could these just compare the elements' references? i.e. `expect(target.nextElementSibling).toBe(plots[0])`

Not super important, just makes things a tiny bit simpler. This understandably apes the existing implementation, so it's not like it comes from nowhere.

Reply:
Good point. I agree with comparing by reference instead of id 👍